internal/storage/binlog_iterator.go

```diff
@@ -46,6 +46,7 @@ type Iterator interface {
 type Value struct {
 	ID        int64
 	PK        PrimaryKey
+	BM25Row   map[int64][]byte
```

Comment:
is this new member necessary?

In `segment_writer.go` we already have `w.bm25Stats` initialized with field id as key, we should be able to:

```go
for fieldID, stats := range w.bm25Stats {
    stats.AppendBytes(v.Value.(map[storage.FieldID]interface{})[fieldID].([]byte)
}
```

in `(w *SegmentWriter) Write`. also we can avoid changes in `serde_events.go`.